app/models/miq_report.rb

```diff
@@ -124,32 +124,6 @@ def self.get_col_info(path)
     }
   end
 
-  def self.display_filter_details(cols, mode)
```

Comment:
sorry, you can't delete and introduce a new method in the same PR.

You'll have to have 3 PRs:

- [ ] add method
- [ ] use that method in the controller repo
- [ ] delete the old method